en/components/select.md

```diff
@@ -240,44 +295,44 @@ Then in your template file you can iterate over these objects and access their p
 
 ### Select In A Form
-<div class="sample-container loading" style="height: 450px;">
-    <iframe id="select-sample-3-iframe" frameborder="0" seamless="" width="100%" height="100%" data-src="{environment:demosBaseUrl}/data-entries/select-sample-3" class="lazyload"></iframe>
+<div class="sample-container loading" style="height: 400px;">
+    <iframe id="select-form-iframe" frameborder="0" seamless="" width="100%" height="100%" data-src="{environment:demosBaseUrl}/data-entries/select-form" class="lazyload"></iframe>
 </div>
 
-<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="select-sample-3-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="select-form-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 
-`igx-select` can also be put inside of a `form` element and in order to do that, you first have to create the template for your control and add the items that it will be displaying:
+`igx-select` can also be used inside of a `form` element:
 
-```html
-<igx-select [(ngModel)]="selected">
-    <igx-select-item value="Orange">Orange</igx-select-item>
-    <igx-select-item value="Apple">Apple</igx-select-item>
-    <igx-select-item value="Banana">Banana</igx-select-item>
-</igx-select>
-```
 
-Another way to do it would be to simply pass in a collection of the items that we want to display to the [*ngForOf*](https://angular.io/api/common/NgForOf) directive:
 ```html
-<igx-select [(ngModel)]="selected">
-    <igx-select-item *ngFor="let item of items" [value]="item">
-        {{item}}
-    </igx-select-item>
-</igx-select>
+<form>
+    <igx-select [(ngModel)]="selected" required>
+        ...
+        <igx-select-item *ngFor="let fruit of fruits" [value]="fruit">
+            {{item}}
+        </igx-select-item>
+        ...
+    </igx-select>
+</form>
 ```
 
 Since we are using two-way binding, your class should look something like this:
-```ts
+```typescript
 export class MyClass {
-  public selected: string = "Apple";
+    public fruits: string[] = ["Orange", "Apple", "Banana", "Mango"];
+    public selected: string = "Apple";;
 }
 ```
 
-You may also notice that in the above sample we have a *prefix* on the input field, this is because `igx-select` supports both prefixes and suffixes. You can read more about them [*here*](https://www.infragistics.com/products/ignite-ui-angular/angular/components/input_group.html).
-- The items' list default exapansion panel arrow uses `IgxSuffix` and it can be changed by the user.
-- If more than one `IgxSuffix` is used, the expansion arrow will be displayed always last.
+### Use with @angular/forms
+`igxSelect` support all of the form directives from the core FormsModule [*NgModel*](https://angular.io/api/forms/NgModel) and [*ReactiveFormsModule*](https://angular.io/api/forms/ReactiveFormsModule) (FormControl, FormGroup, etc.). 
```

Comment:
These should not be italic, wrap them in code blocks instead (e.g. `NgModel` instead of *NgModel*)